sql/core/src/main/scala/org/apache/spark/sql/internal/CatalogImpl.scala

```diff
@@ -138,7 +138,7 @@ class CatalogImpl(sparkSession: SparkSession) extends Catalog {
    */
   @throws[AnalysisException]("table does not exist")
   override def listColumns(tableName: String): Dataset[Column] = {
```

Comment:
Oh, right. We should update here too.
```
/**
 * Returns a list of tables in the specified database.
 * This includes all temporary tables.
 *
 * @since 2.0.0
 */
 @throws[AnalysisException]("database does not exist")
  def listTables(dbName: String): Dataset[Table]
```